### ISSUE TITLE
Visual Studio Removed from Windows Container Image?

### ISSUE THREAD

**Issue by Jonas82** [Microsoft/vsts-image-generation#218]
The build we are running on Windows Container Image recently stopped working with error: 
"Visual Studio version '15.0' not found."
I am no expert in how these build images are made, but from the commit below it seems visual studio build tools was removed!?

https://github.com/Microsoft/vsts-image-generation/commit/d4b6473e29618b0bfc8566e36f4f4454bc20d298#diff-7747d449c67ab9ef713e7393b99dde47

**Comment by lokesh755** [comment]
Yeah.. You can continue to use VS2017 pools for using Visual studio
